hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -176,7 +177,7 @@ newTinyWallet tracer networkId (vk, sk) queryWalletInfo queryEpochInfo = do
   address =
     makeShelleyAddress networkId (PaymentCredentialByKey $ verificationKeyHash vk) NoStakeAddress
 
-  ledgerAddress = toLedgerAddr $ shelleyAddressInEra @Api.Era address
+  ledgerAddress = toLedgerAddr $ shelleyAddressInEra @Api.Era Api.ShelleyBasedEraBabbage address
```

Comment:
This is a bit of an odd piece of code. What we _could_ do here:
* use `mkVkAddress` to easily create an `AddressInEra` and see if we can avoid the `Address ShelleyAddr`, or
* stick with the `Address ShelleyAddr` as this actually does not need an `era` and can always be converted to a ledger `Addr`

Concretely for this PR, I would suggest to avoid mentioning babbage here by:
```suggestion
  ledgerAddress = toLedgerAddr $ shelleyAddressInEra @Api.Era Api.shelleyBasedEra address
```